lms/static/js/verify_student/views/webcam_photo_view.js

```diff
@@ -241,12 +245,18 @@
             );
             $( this.el ).html( renderedHtml );
 
+            var resetBtn = this.$el.find('#webcam_reset_button');
+            var captureBtn = this.$el.find('#webcam_capture_button');
```

Comment:
@AlasdairSwan Nitpick 1 is not fixable because the DOM is not yet loaded until line "$( this.el ).html( renderedHtml );". 2nd issue is fixed and waiting for the test cases to pass for merging